### ISSUE TITLE
cluster deployments dont create any resource

### ISSUE THREAD

**Issue by manishtradebyte** [Altinity/clickhouse-operator#1335]
I was able to create this before but now i dont get any error when i apply it , but no resources are created.

I am trying to deploy this 

```
apiVersion: "clickhouse.altinity.com/v1"
kind: "ClickHouseInstallation"
metadata:
  name: "simple-01"
spec:
  configuration:
    clusters:
      - name: "simple"
    users:
      user2/networks/ip: "::/0"
      user2/password: xccxx
      user2/profile: default
```

when i check operator logs i get this error

1.ClickHouseInstallation: quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'
E0206 18:10:07.674253       1 reflector.go:148] pkg/client/informers/externalversions/factory.go:132: Failed to watch *v1.ClickHouseInstallation: failed to list *v1.ClickHouseInstallation: quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'

**Comment by manishtradebyte** [comment]
I installed 0.23.0 operator on a fresh cluster using k apply -f https://github.com/Altinity/clickhouse-operator/raw/0.23.0/deploy/operator/clickhouse-operator-install-bundle.yaml and it still gives the same error. 

what is the most stable version of the operator?

**Comment by manishtradebyte** [comment]
seems like i was deploying from master . I fixed it by deploying 0.23.0 versoon of operator
